assets/src/components/nav/topNav.tsx

```diff
@@ -49,8 +50,24 @@ const TopNav = (): JSX.Element => {
               placement="bottom"
             >
               <Popover className="c-top-nav__popover">
-                <Popover.Body>
-                  <LoggedInAs email={email} />
+                <Popover.Body className="p-0">
+                  <Dropdown.Menu
+                    show={showUserPopover}
```

Comment:
Non-blocking Nit: 👇 

```suggestion
                    show
```

Since the popover's visibility is already governed by `showUserPopover`, I don't think we also need to have the menu itself be governed that way.

And this makes it easier to extract the `Dropdown.Menu` part to a separate component if we wanted to have it live in Storybook.

Reply:
Good catch, thanks!